plugin.video.dailymotion_com/resources/settings.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
would be nice to migrate the settings to the new format at some point

Reply:
I started doing it but unfortunately noticed that the new format is not supported properly in Leia. With most users still on Leia, i am sticking to the old format for now as it is a single codebase for both Leia and Matrix on my git.
Will certainly do it if Leia nightly fixes it